cloud-core/src/main/java/cloud/commandframework/arguments/parser/ArgumentParseResult.java

```diff
@@ -67,6 +80,18 @@ private ArgumentParseResult() {
         return new ParseSuccess<>(value);
     }
 
+    /**
+     * Create a {@link CompletableFuture future} completed with a successful parse result.
+     *
+     * @param value parsed value
+     * @param <T>   parser value type
+     * @return completed future with successful result
+     * @since 2.0.0
+     */
+    public static <T> @NonNull CompletableFuture<@NonNull ArgumentParseResult<T>> successFuture(final @NonNull T value) {
```

Comment:
```suggestion
    @API(status = API.Status.STABLE, since = "2.0.0")
    public static <T> @NonNull CompletableFuture<@NonNull ArgumentParseResult<T>> successFuture(final @NonNull T value) {
```